diff --git a/STAT/AliTreeTrending.cxx b/STAT/AliTreeTrending.cxx
index 8634b69..380d20c 100644
--- a/STAT/AliTreeTrending.cxx
+++ b/STAT/AliTreeTrending.cxx
@@ -243,14 +243,23 @@ void AliTreeTrending::AppendStatusPad(Float_t padRatio, Float_t bottomMargin, Fl
   // draw original canvas into first pad
   c1->cd(1);
   c1_clone->DrawClonePad();
-  pad1->SetBottomMargin(0.001);
-  pad1->SetRightMargin(rightMargin);
+  
+  AliDrawStyle::SetCssStyle("testStyle",AliDrawStyle::ReadCSSFile("$AliRoot_SRC/STAT/test/alirootTestStyle.css",0));//remove
+  pad1->SetName("Top.class(statusPad)");
+  AliDrawStyle::TPadApplyStyle("testStyle",pad1); //"testStyle" to be changed to fCurrentCSSStyle.Data()
+  ::Info("AppendStatusPad", fCurrentCSSStyle );
+
+//  pad1->SetBottomMargin(0.001);
+//  pad1->SetRightMargin(rightMargin);
   // set up second pad
   c1->cd(2);
-  pad2->SetGrid(3);
-  pad2->SetTopMargin(0);
-  pad2->SetBottomMargin(bottomMargin); // for the long x-axis labels (run numbers)
-  pad2->SetRightMargin(rightMargin);
+//  pad2->SetGrid(3);
+//  pad2->SetTopMargin(0);
+//  pad2->SetBottomMargin(bottomMargin); // for the long x-axis labels (run numbers)
+//  pad2->SetRightMargin(rightMargin);
+  pad2->SetName("Bottom.class(statusPad)");
+  AliDrawStyle::TPadApplyStyle("testStyle",pad2); //"testStyle" to be changed to fCurrentCSSStyle.Data()
+  
   const Int_t nVars = fStatusGraphM->GetYaxis()->GetNbins();
   TGraph* grAxis = (TGraph*) fStatusGraphM->GetListOfGraphs()->At(0);
   Int_t entries = grAxis->GetN();
diff --git a/STAT/test/alirootTestStyle.css b/STAT/test/alirootTestStyle.css
index d811405..060130e 100644
--- a/STAT/test/alirootTestStyle.css
+++ b/STAT/test/alirootTestStyle.css
@@ -15,8 +15,8 @@ TGraph#obj1, TGraph.Status#TPC.QA.dcar_posA_1   {
 
 .deadBand   {
     marker_size:   1,1,1;
-    marker_color:  400,632,416; /* marker_color defenition */
-    line_color:    400,632,416; /* line_color defenition */
+    marker_color:  400,632,416;; /* marker_color defenition */
+    line_color:    2,3,4;/* line_color defenition */
     line_style:    2,3,4;
 }
 
@@ -61,3 +61,16 @@ TH1.Status#obj1, TH1.Warning#obj1, TH1.Warning#obj3   {
     line_color:    57,58,59,60;
     line_style:    61,62,63,64;
 }
+
+.statusPad#Top {
+    bottom_margin:   0.001;
+    right_margin:    0.05;
+}
+
+.statusPad#Bottom{
+    bottom_margin:   0.4;
+    right_margin:    0.05;
+    top_margin:      0.0;  
+    gridX:           3;   
+    gridY:           1;
+}
\ No newline at end of file
